src/ucp/core/ucp_ep.c

```diff
@@ -170,7 +170,7 @@ void ucp_ep_config_key_reset(ucp_ep_config_key_t *key)
     key->dst_md_cmpts     = NULL;
     key->err_mode         = UCP_ERR_HANDLING_MODE_NONE;
     key->flags            = 0;
-    key->wire_version     = UCP_RELEASE_CURRENT;
+    key->peer_version     = UCP_RELEASE_CURRENT;
```

Comment:
IMO dst_version is better because we have dst_md_cmpts, dst_md_index, dst_sys_dev